root/static/scripts/edit/URLCleanup.js

```diff
@@ -257,6 +257,14 @@ const RESTRICTED_LINK_TYPES = _.reduce([
   LINK_TYPES.youtube,
 ], function (result, linkType) {return result.concat(_.values(linkType));}, []);
 
+function reencode_mediawiki_localpart(url) {
```

Comment:
Fixed that, but the tests failed again; apparently another syntax error, unfortunately I can’t tell from what MBJenkins tells me.